ckanext/opendata_theme/extended_themes/bostonma/templates/custom_footer.html

```diff
@@ -7,7 +7,7 @@
         {{ h.opendata_theme_get_footer_data('content_{}'.format(index)) }}
       </div>
     {% endfor %}
-    <div class="col-md-4 span4 footer-links">
+    <!--<div class="col-md-4 span4 footer-links">
```

Comment:
@amrita-suresh Please have the comment start on line 11 `<form id = "foot-letter"` instead of line 10 `<div class="col-md-4 span4 footer-links">`, otherwise there will be spacing issues in the footer.